scalameta/parsers/shared/src/main/scala/scala/meta/parsers/Errors.scala

```diff
@@ -6,7 +6,7 @@ import org.scalameta.data._
 import scala.meta.inputs._
 import scala.meta.internal.inputs._
 
-@root trait Parsed[+T] {
```

Comment:
Not sure why this needed to be marked as ast since this are perfectly fine as case classes.